enterprise/templatetags/enterprise.py

```diff
@@ -0,0 +1,30 @@
+"""
+Template tags for enterprise apps.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()  # pylint: disable=invalid-name
+
+
+@register.simple_tag(name="fa_icon")
+def fa_icon(tags):
+    """
+    Django template tag that returns font awesome icon depending upon message tags.
+
+    Usage:
+        {% fa_icon "success" %}
+    """
+    if 'success' in tags:
```

Comment:
@saleem-latif its true that the message can be only one of success, info, error or warning. But we cannot say about combining some other custom tags and if we decided to show multiple icons then we should return appropriate icons for them. I was suggesting that we should keep things generic. Current template tag accepts multiple tags so its natural to assume it could return multiple icons. This was just my thinking and I am ok with current approach. We can always modify it in future as needed.